client/src/components/Selection.jsx

```diff
@@ -1,25 +1,31 @@
 class Selection extends React.Component {
```

Comment:
-same here, doesn't need to be a class component if you aren't going to keep a state.